src/HuffmanCode.java

```diff
@@ -13,7 +13,7 @@
 public class HuffmanCode {
 	
 	
-	private HuffmanNode overallRoot; // Represents the total number of characters in algorithm
```

Comment:
It is not true that the HuffmanNode is generated from file since all frequency info is lost. I prefer to use "The top node of HuffmanCode" it is the same as your comment in 20 Questions

Reply:
How about "Represents the algorithm to compress/decompress message"

Reply:
I have "stores huffman coding algorithm" as the comment rn